crcsim/experiment/prepare.py

```diff
@@ -144,8 +165,13 @@ def create_scenarios() -> List:
     low_surveillance_freq_mild = 10
     low_surveillance_freq_severe = 2
     low_surveillance_end_age = 80
+    start_age = 50
+    end_age = 75
     scenarios = []
 
+    fifty_percent_compliance = {year: 0.5 for year in range(start_age, end_age + 1)}
+    twenty_percent_compliance = {year: 0.2 for year in range(start_age, end_age + 1)}
```

Comment:
I think you're on the right track here for the random compliance scenarios, but again, this needs to be formatted to adjust the existing conditional compliance parameters